packages/api-client/src/store/workspace.ts

```diff
@@ -197,6 +197,7 @@ const createParamInstance = (param: OpenAPIV3_1.ParameterObject) =>
           : '',
     description: param.description,
     required: param.required,
+    enabled: !!param.required,
```

Comment:
```suggestion
    enabled: Boolean(param.required),
```
nit pick but this one is a bit more succinct

Reply:
but also shouldn't even be needed?

Reply:
woah ive never seen this before 🤯 yeah i think youre right, lemme remove

Reply:
well, it could be undefined 
![image](https://github.com/user-attachments/assets/d536973a-ddc0-47db-b8cd-071f5ee095a3)